pkg/machines/vmdiskstab.jsx

```diff
@@ -120,22 +120,19 @@ const VmDisksTab = ({ vm, provider }) => {
         supported, but not available yet ~ will be read soon
         unsupported ~ failed attempt to read (old libvirt)
      */
-    let areDiskStatsSupported = true;
-    let showDiskStatsColumns = false;
+    let areDiskStatsSupported = false;
     if (vm.disksStats) {
-        const isNaNFound = Object.getOwnPropertyNames(vm.disksStats).some(target => { // look for NaN value
+        const isNonNaNFound = Object.getOwnPropertyNames(vm.disksStats).some(target => { // look for non-NaN value (means disk stats at least partially read)
             if (!vm.disksStats[target]) {
                 return false; // not yet retrieved, can't decide about disk stats support
             }
-            showDiskStatsColumns = true; // at least one disk stats read attempt finished
-            return isNaN(vm.disksStats[target].capacity); // if true, disk stats read attempt executed, but failed
+            return !isNaN(vm.disksStats[target].capacity) || !isNaN(vm.disksStats[target].allocation);
         });
-        areDiskStatsSupported = !isNaNFound;
-        showDiskStatsColumns = showDiskStatsColumns && areDiskStatsSupported;
+        areDiskStatsSupported = !!isNonNaNFound;
```

Comment:
why not assign this directly here if `isNonNaNFound` isn't used in other places anyway...